snakemake_interface_storage_plugins/io.py

```diff
@@ -13,7 +13,7 @@
     \{
         (?=(   # This lookahead assertion emulates an 'atomic group'
                # which is required for performance
-            \s*(?P<name>\w+)                    # wildcard name
+            \s*(?P<name>(wildcards\.\w+|\w+))   # wildcard name
```

Comment:
What if we just allowed for dots in `name`?
```suggestion
            \s*(?P<name>[.\w]+)   # wildcard name
```